userspace/falco/app/actions/print_support.cpp

```diff
@@ -32,28 +38,84 @@ static std::string read_file(const std::string &filename)
 	return str;
 }
 
+#ifndef _WIN32
+static int get_unix_sysinfo(nlohmann::json &support)
+{
+	struct utsname sysinfo;
+	if(uname(&sysinfo) != 0)
+	{
+		return -1;
+	}
+
+	support["system_info"]["sysname"] = sysinfo.sysname;
+	support["system_info"]["nodename"] = sysinfo.nodename;
+	support["system_info"]["release"] = sysinfo.release;
+	support["system_info"]["version"] = sysinfo.version;
+	support["system_info"]["machine"] = sysinfo.machine;
+	return 0;
+}
+#endif
+
+#ifdef _WIN32
+static int get_win32_sysinfo(nlohmann::json &support)
```

Comment:
Why not call both of these with the same name? That way we don't need any ifdef when invoking them.